airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -381,6 +383,7 @@ def __init__(
         self.deferrable = deferrable
         self.poll_interval = poll_interval
         self.remote_pod: k8s.V1Pod | None = None
+        self.log_pod_spec_on_failure = log_pod_spec_on_failure
```

Comment:
Where is this used?

Reply:
here: https://github.com/apache/airflow/pull/31595/files#diff-93707635f12195e7b94d14cf99675542592a28a6eb552895fb3ff37d42713740R704

Reply:
Maybe Ephraim meant to ask where is the function getting called and the value being set to False to disable the pod spec in the error.

Reply:
Indeed, there is a typo in the line where it is used ([fix](https://github.com/apache/airflow/pull/31595#discussion_r1209332736))

Reply:
sure, thx for your fix